library/core/src/any.rs

```diff
@@ -673,7 +673,7 @@ impl hash::Hash for TypeId {
 #[stable(feature = "rust1", since = "1.0.0")]
 impl fmt::Debug for TypeId {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
-        f.debug_tuple("TypeId").field(&self.as_u128()).finish()
+        write!(f, "TypeId({:#034x})", self.as_u128())
```

Comment:
Shouldn't this be 32 instead of 34?

Reply:
The number has to include the leading `0x` when used with `#`. Which always feels pretty weird to me but is apparently correct (e.g. the example in the top post)